### DISCUSSION TITLE
Review and Comments: 02-component-descriptor.md

### DISCUSSION THREAD

**Discussion by jensh007** [open-component-model/ocm-spec#13] [General]
> ## Repository Contexts
> ... The repositoryContexts are usually not specified manually in the Component Descriptor, but are rather set automatically when a component version is uploaded to a Component Repository.

I suggest to remove this sentence. It is not relevant for the spec if this is manually or automatically created. If this is part of the spec it must be described what "automatically" means (who and when)?

**Comment by achimweigel** [comment]
I thought the last entry of repositoryContexts is set by the component repo when uploading a component descriptor. We must somehow define that a component descriptor could not be uploaded with a wrong last entry. Otherwise we would have somehow inconsistent data.
